cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -1338,7 +1338,7 @@ impl ManagedEnvironment {
         // ensure_locked is avoiding locking of v0 manifests,
         // but we don't need to support pushing old manifests.
         core_environment
-            .lock(flox)
+            .ensure_locked(flox)
```

Comment:
nonblocking: why would we change this? If we do change it comment needs update

Reply:
pushing of v0 lockfiles failed and was required for a few tests iirc, and not supporting push at all when it seems to only be a change of lock -> ensure locked seemed reasonable enough. Esp. since this is not locking pkgdb either, just allowing to push _existing_ and _locked_ envs to floxhub.

I can break out a "fake push" if its important to not support pushing of v0, but i'd rather amend the comment instead.

e.g. here:
https://github.com/flox/flox/blob/a949f96ce596aa468ac8cfaf96ea6187a8b09518/cli/tests/pull.bats#L358-L363

https://github.com/flox/flox/blob/a949f96ce596aa468ac8cfaf96ea6187a8b09518/cli/tests/environment-managed.bats#L60-L70

https://github.com/flox/flox/blob/a949f96ce596aa468ac8cfaf96ea6187a8b09518/cli/tests/environment-remote.bats#L67-L84

Reply:
changed the comment in db1a6684576b54fd0c6a9d42680d0fc9865dd622

Reply:
That makes sense, not worth the effort of changing the tests 👍